carma/launch/plugins.launch.py

```diff
@@ -107,12 +107,12 @@ def generate_launch_description():
                 ]
             ),
             ComposableNode(
-                package='stop_and_wait_plugin',
-                plugin='stop_and_wait_plugin::StopandWaitNode',
-                name='stop_and_wait_plugin',
+                package='route_following_plugin',
```

Comment:
looks like route following and inlane-cruising are still in the came container

Reply:
seperated them